misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/Main.java

```diff
@@ -50,11 +50,19 @@ public static void main(String[] args) throws IOException {
             //convert the json string back to object
             Gson gson = new Gson();
             TestSuite response = gson.fromJson(br, TestSuite.class);
-            response.setModuleName(args[args.length - 1]);
+            response.setModuleName(filterModuleName(args[args.length - 1]));
             startTestSuit(Paths.get(response.getSourceRootPath()), response, jsonTmpSummaryPath);
         }
     }
 
+    private static String filterModuleName(String argument) {
+        //Empty string needs to be passed for default module
+        if (argument.startsWith("\"") && argument.endsWith("\"") && argument.length() == 2) {
```

Comment:
Shall we refactor the validation as `if (argument != null && !argument.trim().isEmpty())`